### PR TITLE
cli package revisited.

### PR DESCRIPTION
**Pull request by protich**
Package the correct scp sub-directories  -> css images js
Exclude ost-config.php
Use the correct package name

### CODE REVIEW

File: setup/cli/package.php [hunk 1]
```diff
@@ -89,22 +89,23 @@ function package($pattern, $destination, $recurse=false, $exclude=false) {
...
 
 # Load in the scripts
 mkdir("$stage_path/scripts/");
 package("setup/scripts/*", "scripts/", -1, "*stage");
 
 # Load the heart of the system
-package("include/*.php", "upload/include", -1);
-# And the sql patches
-package("include/upgrader/*.sql", "upload/include/upgrader", -1);
+package("include/{,.}*", "upload/include", -1, array('*ost-config.php', '*.sw[a-z]'));
```
Comment (greezybacon) [inline setup/cli/package.php:108]: What's up with the commas? Do we have files with commas in the filenames?

Comment (protich) [inline setup/cli/package.php:108]: Separator - basically means "any, ." - glob doesn't like spaces so you can't do " , ."

**Comment by greezybacon** [comment]
Have you done a complete install via FTP with the ZIP package emitted?

**Comment by protich** [comment]
I did yesterday and it didn't go well - hence the fixes here. I'll be doing some more testing today.

diff'ing stage/upload with the live code helped - we should probably add it to packager.

**Pull request merged**